eng/Versions.props

```diff
@@ -7,30 +7,30 @@
     <PreReleaseVersionLabel>beta3</PreReleaseVersionLabel>
     <!-- CoreFX -->
     <MicrosoftExtensionsFileSystemGlobbingVersion>2.0.0</MicrosoftExtensionsFileSystemGlobbingVersion>
-    <SystemCollectionsImmutableVersion>1.3.1</SystemCollectionsImmutableVersion>
+    <SystemCollectionsImmutableVersion>5.0.0</SystemCollectionsImmutableVersion>
     <SystemReflectionMetadataVersion>1.4.2</SystemReflectionMetadataVersion>
-    <SystemValueTupleVersion>4.4.0</SystemValueTupleVersion>
+    <SystemValueTupleVersion>4.5.0</SystemValueTupleVersion>
     <SystemIOCompressionVersion>4.3.0</SystemIOCompressionVersion>
     <SystemIOPackagingVersion>4.5.0</SystemIOPackagingVersion>
     <SystemNetHttpVersion>4.3.4</SystemNetHttpVersion>
     <SystemTextRegularExpressions>4.1.0</SystemTextRegularExpressions>
     <!-- VS -->
-    <EnvDTEVersion>8.0.0</EnvDTEVersion>
-    <MicrosoftInternalVisualStudioShellInterop100DesignTimeVersion>10.0.30319</MicrosoftInternalVisualStudioShellInterop100DesignTimeVersion>
-    <MicrosoftInternalVisualStudioShellInterop110DesignTimeVersion>11.0.61030</MicrosoftInternalVisualStudioShellInterop110DesignTimeVersion>
-    <MicrosoftVisualStudioShell150Version>15.0.26606</MicrosoftVisualStudioShell150Version>
-    <MicrosoftVisualStudioSettings150Version>15.0.26606</MicrosoftVisualStudioSettings150Version>
-    <MicrosoftBuildUtilitiesCoreVersion>15.1.548</MicrosoftBuildUtilitiesCoreVersion>
+    <EnvDTEVersion>17.0.0-previews-2-31429-540</EnvDTEVersion>
```

Comment:
💡 Rather than update all the individual assemblies, you can just reference Microsoft.VisualStudio.SDK.